<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="67" started-at="2020-08-14T12:55:36Z" finished-at="2020-08-14T12:55:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="67" started-at="2020-08-14T12:55:36Z" finished-at="2020-08-14T12:55:36Z">
      <class name="com.shopcart.qa.testcases.AddnewManufactureAddressTest">
        <test-method status="FAIL" signature="Setup()[pri:0, instance:com.shopcart.qa.testcases.AddnewManufactureAddressTest@194d431]" name="Setup" is-config="true" duration-ms="0" started-at="2020-08-14T18:25:36Z" finished-at="2020-08-14T18:25:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.shopcart.qa.base.TestBase.initialization(TestBase.java:67)
	at com.shopcart.qa.testcases.AddnewManufactureAddressTest.Setup(AddnewManufactureAddressTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="SKIP" signature="checkaddnew()[pri:0, instance:com.shopcart.qa.testcases.AddnewManufactureAddressTest@194d431]" name="checkaddnew" duration-ms="1" started-at="2020-08-14T18:25:36Z" finished-at="2020-08-14T18:25:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkaddnew -->
      </class> <!-- com.shopcart.qa.testcases.AddnewManufactureAddressTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
